x-pack/plugins/apm/server/routes/source_maps/route.ts

```diff
@@ -169,6 +192,85 @@ const uploadSourceMapRoute = createApmServerRoute({
   },
 });
 
+const uploadAndroidMapRoute = createApmServerRoute({
```

Comment:
Could you please add docs for this one? (Example: https://www.elastic.co/guide/en/kibana/current/apm-api.html)